File: WEB/src/Entity/Produit.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Produit
 *
 * @ORM\Table(name="produit", indexes={@ORM\Index(name="fk_idcatedel", columns={"idCategorie"})})
 * @ORM\Entity
 */
class Produit
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;


    /**
     * @var string|null
     *
     * @ORM\Column(name="libelle", type="string", length=20, nullable=true)
     */
    private $libelle;

    /**
     * @var float|null
     *
     * @ORM\Column(name="prix", type="float", precision=10, scale=0, nullable=true)
     */
    private $prix;

    /**
     * @var string|null
     *
     * @ORM\Column(name="description", type="string", length=20, nullable=true)
     */
    private $description;

    /**
     * @var int
     *
     * @ORM\Column(name="qt", type="integer", nullable=false)
     */
    private $qt;

    /**
     * @var string|null
     *
     * @ORM\Column(name="img", type="string", length=255, nullable=true)
     */
    private $img;

    /**
     * @var int
     *
     * @ORM\Column(name="idCategorie", type="integer", nullable=false)
     */
    private $idcategorie;

    public function getId()
    {
        return $this->id;
    }

    public function setId($id)
    {
        $this->id = $id;
    }

    public function getLibelle()
    {
        return $this->libelle;
    }


    public function setLibelle($libelle)
    {
        $this->libelle = $libelle;
    }


    public function getPrix()
    {
        return $this->prix;
    }


    public function setPrix($prix)
    {
        $this->prix = $prix;
    }


    public function getDescription()
    {
        return $this->description;
    }

    public function setDescription($description)
    {
        $this->description = $description;
    }

    public function getQt()
    {
        return $this->qt;
    }


    public function setQt($qt)
    {
        $this->qt = $qt;
    }

    public function getImg()
    {
        return $this->img;
    }

    public function setImg($img)
    {
        $this->img = $img;
    }

    public function getIdcategorie()
    {
        return $this->idcategorie;
    }

    public function setIdcategorie($idcategorie)
    {
        $this->idcategorie = $idcategorie;
    }
}

File: Mobile/src/Entities/Produit.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

/**
 *
 * @author ASUS
 */
public class Produit {
    protected int id;
    protected int idCategorie;
    protected String nomcategorie;
    protected String libelle;
    protected float prix;
    protected String description;
    protected int qt;
    protected String img;
    protected boolean testimg;
    protected int rating;
    public Produit(){
    }
    
    public Produit(int id,int idCategorie){
        this.id = id;
        this.idCategorie = idCategorie;
    }
    
    public Produit(String libelle,float prix,String description){
        this.libelle = libelle;
        this.prix = prix;
        this.description = description;
    }
    
    public Produit(int id,int idCategorie,String libelle,float prix,String description){
        this.id = id;
        this.idCategorie = idCategorie;
        this.libelle = libelle;
        this.prix = prix;
        this.description = description;
    }
    
    public Produit(int id,int idCategorie,String libelle,float prix,String description,int qt,String img){
        this.id = id;
        this.idCategorie = idCategorie;
        this.libelle = libelle;
        this.prix = prix;
        this.description = description;
        this.qt = qt;
        this.img = img;
    }
    
    public Produit(int id,int idCategorie,String libelle,float prix,String description,int qt,String img,boolean testimg){
        this.id = id;
        this.idCategorie = idCategorie;
        this.libelle = libelle;
        this.prix = prix;
        this.description = description;
        this.qt = qt;
        this.img = img;
    }
    
    public int getId(){
        return this.id;
    }
    
    public void setId(int id){
        this.id = id;
    }
    
    public int getIdCategorie(){
        return this.idCategorie;
    }
    
    public void setIdCategorie(int idCategorie){
        this.idCategorie = idCategorie;
    }
    
    public String getNomcategorie(){
        return this.nomcategorie;
    }
    
    public void setnomCategorie(String nomcategorie){
        this.nomcategorie = nomcategorie;
    }
    
    public String getLibelle(){
        return this.libelle;
    }
    
    public void setLibelle(String libelle){
        this.libelle = libelle;
    }
    
    public float getPrix(){
        return this.prix;
    }
    
    public void setPrix(float prix){
        this.prix = prix;
    }
    
    public String getDescription(){
        return this.description;
    }
    
    public void setDescription(String description){
        this.description = description;
    }
    
    public int getQt(){
        return this.qt;
    }
    
    public void setQt(int qt){
        this.qt = qt;
    }
    
    public String getImg(){
        return this.img;
    }
    
    public void setImg(String img){
        this.img = img;
    }
    
    public boolean getTestimg(){
        return this.testimg;
    }
    
    public int getRating(){
        return this.rating;
    }
    
    public void setRating(int rating){
        this.rating = rating;
    }
    
   /* @Override
    public String toString(){
        return "\nId: " + this.id +"\nId Categorie: "+this.idCategorie+"\nLibelle: " + this.libelle+ "\nPrix: "+this.prix+"\ndescription:"+this.description+"\n";
    }
    */
    @Override
    public String toString(){
        return String.valueOf(this.id);
    }
}
